recipe/meta.yaml

```diff
@@ -1,70 +1,77 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.8.3.1" %}
+{% set sha256 = "4b0f5a5b4828915d973d3cbb86372aea9207e7b60fed912045d69242a74407f6" %}
+
 package:
-    name: ulmo
-    version: 0.8.2
+  name: ulmo
+  version: {{ 0.8.3.1 }}
```

Comment:
nah, just wanted to make sure I understood what the policy was. I'm already
running tests upstream for ulmo on travis and appveyor.

- dharhas

On Mon, Jul 18, 2016 at 5:05 PM Filipe <notifications@github.com> wrote:

> In recipe/meta.yaml
> <https://github.com/conda-forge/ulmo-feedstock/pull/2#discussion_r71239520>
> :
>
> >  package:
> > -    name: ulmo
> > -    version: 0.8.2
> > +  name: ulmo
> > +  version: {{ 0.8.3.1 }}
>
> That is correct. We can run the tests, but usually we trust they are run
> upstream and that we only need to check if the package can be "used."
>
> The main reason is to avoid an extra burden on the CIs, but if the tests
> are light weight and run under the CIs time limit feel free to add them
> here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/ulmo-feedstock/pull/2/files/e0cd0250b2306392b1553ce5830c44d540f5502c#r71239520>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAiQlYyiq8dTxg5Bwn9OA7ENXaClr4fXks5qW_iPgaJpZM4JPKIl>
> .
>